.circleci/config.yml

```diff
@@ -0,0 +1,305 @@
+version: 2
+
+references:
+  container_config: &container_config
+    docker:
+      - image: aetrnty/builder
+        user: builder
+    working_directory: ~/epoch
+
+  rebar_cache_key: &rebar_cache_key rebar-cache-{{ checksum "rebar.lock" }}-{{ checksum "rebar.config" }}
+  restore_rebar_cache: &restore_rebar_cache
+    restore_cache:
+      key: *rebar_cache_key
+
+  restore_build_cache: &restore_build_cache
+    restore_cache:
+      key: build-cache-v1
+
+  packages_workspace: &packages_workspace /tmp/packages
+  build_package: &build_package
+    run:
+      name: Build Package Tarball
+      environment:
+        PACKAGES_DIR: *packages_workspace
+      command: |
+        if [ "$(uname -s)" == "Darwin" ]; then
+            PKG_SUFFIX="osx-$(sw_vers -productVersion)"
+        elif [ "$(uname -s)" == "Linux" ]; then
+            PKG_SUFFIX="linux"
+        fi
+        make prod-package
+        mkdir ${PACKAGES_DIR:?}
+        cp _build/prod/rel/epoch/epoch-$(cat VERSION).tar.gz ${PACKAGES_DIR:?}/epoch-$(cat VERSION)-${PKG_SUFFIX}.tar.gz
```

Comment:
Could you please prefer mv to cp so avoid unexpected archives around?

Reply:
While it looks more natural to me to keep the rebar build immutable from "outside" and just "pick" what's needed, I applied your request because probably you have better reasoning. a7607ac